src/components/CustomAnimatedModal.tsx

```diff
@@ -7,6 +7,7 @@ import { RequestSwapStack } from '../navigation/RequestSwap/RequestSwapStack';
 interface CustomAnimatedModalProps {
     isVisible: boolean;
     closeModal: () => void;
+    children?;
```

Comment:
does it need to be optional?

Reply:
i mean it'll work if no children are passed in haha so in that sense its optional